source/envs/breakout.py

```
@@ -0,0 +1,877 @@
+                # 가장 높은 reward를 가진 모델 선택
+                best_model = None
+                best_reward = -float("inf")
+                for fname in model_files:
+                    try:
+                        # 파일명에서 reward 추출 (언더바로 변경된 형태)
+                        reward_part = fname.split("_reward_")[1].replace(".pth", "")
+                        reward = float(reward_part.replace("_", "."))
+                        if reward > best_reward:
+                            best_reward = reward
+                            best_model = fname
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)